build.sh

```diff
@@ -17,7 +17,7 @@ docker build -t gradle -f Dockerfile.gradle .
 docker build -t koupler_build -f Dockerfile.build .
 ID="$(docker run -d \
 	-v "${HOME}/.gradle/caches:/root/.gradle/caches" \
-	koupler_build build copyRuntimeLibs batchZip)"
+	koupler_build dist)"
```

Comment:
I had a reason for not using `dist` before... I don't remember what it was, but is there a particular reason you switched to it?